### PR TITLE
Add schema based on diagram

### PR DESCRIPTION
**Pull request by awais-amjed**
In this Pull Request, the following changes were introduced:

- Created a database based on the Schema Diagram Provided ✅

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,63 @@
+CREATE TABLE patients (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(255),
+    date_of_birth DATE,
+    PRIMARY KEY (id)
+);
+
```
Comment (Meltrust) [inline schema_based_on_diagram.sql:7]: - [ ] According to the project requirements, all Foreign Keys need to have indexes created:

```sql
CREATE INDEX ON medical_histories (patient_id);
CREATE INDEX ON invoices (medical_history_id);
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
CREATE INDEX ON medical_histories_has_treatments (treatment_id);
```

Please, add them so the database's performance improves! 💪 

- [ ] All tables need a primary key set, just as you did in the `patients` table.

**Review by Meltrust — APPROVED** [review]
Hi @awais-amjed,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice start on the tables✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema_based_on_diagram.sql (hunk 1 above)
Comment (awais-amjed) [reply schema_based_on_diagram.sql:7]: Thanks a lot for pointing this out. I will add these.

**Review by EricMbouwe — COMMENTED** [review]
### Hi @awais-amjed, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**